packages/app/src/components/catalog/EntityPage/defaultTabs.tsx

```diff
@@ -14,66 +14,81 @@ import { OverviewTabContent } from './OverviewTabContent';
  */
 export const defaultTabs: Record<
   string,
-  Omit<DynamicEntityTabProps, 'if' | 'children' | 'path'>
+  { title: string; titleKey?: string; mountPoint: string; priority?: number }
```

Comment:
Please extend the DynamicEntityTabProps type instead and restore the Omit type here.